core/src/main/scala/org/bitcoins/core/wallet/utxo/InputSigningInfo.scala

```diff
@@ -88,33 +88,47 @@ case class ScriptSignatureParams[+InputType <: InputInfo](
     this.copy(inputInfo = func(this.inputInfo))
   }
 
+  private var _maxScriptSigAndWitnessWeight: Option[(Long, Long)] = None
```

Comment:
What's the reason that this needs a `var`? Isnt' the computation relatively inexpensive so we can just compute it everytimg `maxScriptSigAndWitnessWeight()` is called?

Reply:
it is not inexpensive is the reason. On mac CI this can take like 10 seconds.